.github/workflows/ql-for-ql-build.yml

```
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Find codeql
         id: find-codeql
-        uses: github/codeql-action/init@erik-krogh/ql
+        uses: github/codeql-action/init@erik-krogh/newQl
```

Is anything preventing us from doing a (force) push to erik-krogh/ql instead of creating erik-krogh/newQl? Then we can keep the `ql` branch name here...

---

It's to ensure that the branch bump and the workflow change happen at the same time.

---

I could force push on the old brach, but that could break existing PRs.
(I've already broken all the PRs once).

---

If that is a general concern, then we should change `@erik-krogh/newQl` to the tip:

```suggestion
        uses: github/codeql-action/init@aa93aea877e5fb8841bcb1193f672abf6e9f2980
```

Otherwise, we will have to make `erik-krogh/newQl-2` next time.

---

https://github.com/github/codeql-action/tree/erik-krogh/newQl